### DISCUSSION TITLE
Jaime Aleman

### DISCUSSION THREAD

**Discussion by jjaimealeman** [warengonzaga/css-text-portrait-builder#30] [Showcase]
http://striped-joke.surge.sh/

My first build. I edited the line-height and font-size.

Image from one of the many 240z accounts I follow on Instagram.

Content from https://en.wikipedia.org/wiki/Nissan_S30

**Comment by warengonzaga** [comment]
Hi @jjaimealeman you use the builder with this?

Looks like you forgot to edit the `name` object in `config.json` file.

**Comment by jjaimealeman** [comment]
> `name` object

Thanks man, fixed ;)
